KubeArmor/enforcer/bpflsm/bpffs.go

```diff
@@ -0,0 +1,222 @@
+package bpflsm
```

Comment:
License header is not present.

Please add credits for Cilium folks, since this code is heavily inspired by it.

Wondering if we could just import the specific package from cilium without having to cut-paste the code.

Reply:
> Please add credits for Cilium folks, since this code is heavily inspired by it.

Yep, was going to do that along with the gosec changes.
> Wondering if we could just import the specific package from cilium without having to cut-paste the code.

They have some logs and paths put in according to Cilium, so I don't think that we can import directly : (